src/main/java/com/google/sps/database/DatabaseImpl.java

```diff
@@ -20,6 +20,10 @@
 
     
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    private static final String vKind = "video";
+    private static final String mKind = "metadata";
+    private static final String cKind = "caption";
+    private static final String tKind = "timestamps";
```

Comment:
just being picky, use V_KIND instead of vKind to conform to Java naming of constants

Reply:
you right i forgot about that convention, thanks